sql/analyzer/tables.go

```diff
@@ -112,6 +112,26 @@ func getTable(node sql.Node) sql.Table {
 	return table
 }
 
+// Finds first unresolved table node that is a descendant of the node given
+func hasTable(name string, node sql.Node) bool {
+	var found bool
+	transform.Inspect(node, func(node sql.Node) bool {
+		switch n := node.(type) {
+		case *plan.UnresolvedTable:
+			found = found ||
+				name == n.Name()
```

Comment:
Case insensitive match here plz

Reply:
same thing as above, mysql appears to differentiate case-sensitive names inside recursive ctes